src/aiperf/gpu_telemetry/manager.py

```
@@ -203,6 +203,17 @@ async def _configure_pynvml_collector(self) -> None:
                 self._collectors[PYNVML_SOURCE_IDENTIFIER] = collector
                 self._collector_id_to_url[collector_id] = PYNVML_SOURCE_IDENTIFIER
                 self.debug("GPU Telemetry: pynvml collector configured successfully")
+
+                self.info("GPU Telemetry: Capturing baseline metrics...")
+                try:
+                    await collector.initialize()
+                    await collector.collect_and_process_metrics()
+                    self.debug("GPU Telemetry: Captured baseline from pynvml")
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)